lib/trino-filesystem-s3/src/main/java/io/trino/filesystem/s3/S3FileSystemConfig.java

```diff
@@ -397,6 +398,19 @@ public S3FileSystemConfig setUseWebIdentityTokenCredentialsProvider(boolean useW
         return this;
     }
 
+    public boolean isAnonymousAccess()
+    {
+        return anonymousAccess;
```

Comment:
Looking at:

- https://trino.io/docs/current/object-storage/file-system-azure.html#access-key-authentication
- https://trino.io/docs/current/object-storage/file-system-gcs.html#authentication

I see the use of an "authentication type" field to distinguish between the many potential ways to configure the authentication for s3 access.
This can be done though EVENTUALLY in a follow-up PR.

Or if we decide to add it - then we'd be working with

```
s3.auth-type=ANONYMOUS
```